### ISSUE TITLE
Failure in t/spec/S17-procasync/stress.t

### ISSUE THREAD

**Issue by zoffixznet** [rakudo/rakudo#2203]
This is fairly fresh; started happening in at most the past 7 days. Potentially in [this bump](https://github.com/rakudo/rakudo/commit/0b3fc077e5a196f60553cb85f1dc0d7f05eee25c), because IIRC I saw this file fail during bumps, but I assumed it was a flopper (this file is a regular flopper).

I get this failure consistently:

```
$ ./perl6 -v
This is Rakudo version 2018.06-422-g36890f1 built on MoarVM version 2018.06-414-gb7ffccf
implementing Perl 6.c.


cpan@perlbuild4~/R/rakudo (master)$ z t t/spec/S17-procasync/stress.t
t/spec/S17-procasync/stress.t .. 
1..23
/bin/cat: packages/README: No such file or directory
/bin/cat: This: No such file or directory
/bin/cat: directory: No such file or directory
/bin/cat: exists: No such file or directory
/bin/cat: to: No such file or directory
/bin/cat: hold: No such file or directory
/bin/cat: files: No such file or directory
/bin/cat: tested: No such file or directory
/bin/cat: in: No such file or directory
/bin/cat: S10-packages: No such file or directory
/bin/cat: so: No such file or directory
/bin/cat: that: No such file or directory
/bin/cat: there: No such file or directory
/bin/cat: isn't: No such file or directory
/bin/cat: a: No such file or directory
/bin/cat: hyphen: No such file or directory
/bin/cat: in: No such file or directory
/bin/cat: the: No such file or directory
/bin/cat: filename: No such file or directory
/bin/cat: of: No such file or directory
/bin/cat: the: No such file or directory
/bin/cat: .pm: No such file or directory
/bin/cat: file.: No such file or directory
/bin/cat: This: No such file or directory
/bin/cat: way,: No such file or directory
/bin/cat: we: No such file or directory
/bin/cat: can: No such file or directory
/bin/cat: just: No such file or directory
/bin/cat: use: No such file or directory
/bin/cat: packages::Whatever;: No such file or directory
/bin/cat: instead: No such file or directory
/bin/cat: of: No such file or directory
/bin/cat: needing: No such file or directory
/bin/cat: to: No such file or directory
/bin/cat: quote: No such file or directory
/bin/cat: the: No such file or directory
/bin/cat: package: No such file or directory
/bin/cat: name: No such file or directory
/bin/cat: all: No such file or directory
/bin/cat: the: No such file or directory
/bin/cat: time,: No such file or directory
/bin/cat: or: No such file or directory
/bin/cat: add: No such file or directory
/bin/cat: the: No such file or directory
/bin/cat: proper: No such file or directory
/bin/cat: directory: No such file or directory
/bin/cat: to: No such file or directory
/bin/cat: the: No such file or directory
/bin/cat: Perl: No such file or directory
/bin/cat: 6: No such file or directory
/bin/cat: equivalent: No such file or directory
/bin/cat: of: No such file or directory
/bin/cat: @INC.: No such file or directory
/bin/cat: *****************************************************************************: No such file or directory
/bin/cat: **: No such file or directory
/bin/cat: WARNING:: No such file or directory
/bin/cat: This: No such file or directory
/bin/cat: file: No such file or directory
/bin/cat: can: No such file or directory
/bin/cat: be: No such file or directory
/bin/cat: used: No such file or directory
/bin/cat: by: No such file or directory
/bin/cat: some: No such file or directory
/bin/cat: tests: No such file or directory
/bin/cat: as: No such file or directory
/bin/cat: an: No such file or directory
/bin/cat: arbitrary: No such file or directory
/bin/cat: file: No such file or directory
/bin/cat: to: No such file or directory
/bin/cat: read: No such file or directory
/bin/cat: **: No such file or directory
/bin/cat: arbitrary: No such file or directory
/bin/cat: data: No such file or directory
/bin/cat: from: No such file or directory
/bin/cat: and: No such file or directory
/bin/cat: rely: No such file or directory
/bin/cat: on: No such file or directory
/bin/cat: it: No such file or directory
/bin/cat: being: No such file or directory
/bin/cat: located: No such file or directory
/bin/cat: here,: No such file or directory
/bin/cat: under: No such file or directory
/bin/cat: this: No such file or directory
/bin/cat: name.: No such file or directory
/bin/cat: **: No such file or directory
/bin/cat: If: No such file or directory
/bin/cat: changing: No such file or directory
/bin/cat: the: No such file or directory
/bin/cat: name: No such file or directory
/bin/cat: or: No such file or directory
/bin/cat: location,: No such file or directory
/bin/cat: ensure: No such file or directory
/bin/cat: all: No such file or directory
/bin/cat: tests: No such file or directory
/bin/cat: still: No such file or directory
/bin/cat: pass.: No such file or directory
/bin/cat: *****************************************************************************: No such file or directory
The spawned command '/bin/cat' exited unsuccessfully (exit code: 1)
  in block  at t/spec/S17-procasync/stress.t line 20
  in block <unit> at t/spec/S17-procasync/stress.t line 9

# Looks like you planned 23 tests, but ran 0
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 23/23 subtests 

Test Summary Report
-------------------
t/spec/S17-procasync/stress.t (Wstat: 65280 Tests: 0 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 23 tests but ran 0.
Files=1, Tests=0,  0 wallclock secs ( 0.02 usr  0.00 sys +  0.69 cusr  0.06 csys =  0.77 CPU)
Result: FAIL
The spawned command 't/fudgeandrun' exited unsuccessfully (exit code: 1)
  in block <unit> at /home/cpan/zscript/bin/z line 154

cpan@perlbuild4~/R/rakudo (master)$ 
```

**Comment by jnthn** [comment]
I somewhat suspected 925c1134336687f5 but reverting that didn't help.

**Comment by jnthn** [comment]
Golfed to:

```
constant $read-file = $?FILE.IO; say <<foo $read-file>>
```

Bissected to:

https://github.com/rakudo/rakudo/commit/3ae97e5f38e85a5db76053ed2c7757c59a22f73d

**Comment by lizmat** [comment]
Fixed with e5568aa502

**Comment by zoffixznet** [comment]
Closing without any new tests, as the failure was detected by an already-existing test.
